specification/machinelearning/resource-manager/readme.typescript.md

```diff
@@ -13,8 +13,8 @@ batch:
     output-folder: $(typescript-sdks-folder)/packages/@azure/arm-commitmentplans
   - package-webservices: true
     package-name: "@azure/arm-webservices"
-    output-folder: $(typescript-sdks-folder)/packages/@azure/arm-webservices
+    output-folder: $(typescript-sdks-folder)/sdk/webservices/arm-webservices
```

Comment:
Should these be under a machinelearning service folder?

Reply:
I wonder if the other machinelearning* folders should also be included.

Reply:
yes putting under macinelearning folder, not sure about others